test/distance.test.js

```diff
@@ -0,0 +1,23 @@
+const { distance } = require("../lib");
+
+describe("ASCIIToString", () => {
```

Comment:
Cool! This is a typo though. Should be describe("distance"

Reply:
Argh, missed my copy and paste